app/assets/stylesheets/rfps.css.scss

```diff
@@ -30,7 +30,7 @@
 
 ol.form-list {
   padding-right: 50px;
-  padding-top: 40px; 
```

Comment:
@JohnJasonFisher 
What's up with you text editor? This auto-remove whitespace makes PR's too big.